src/main/groovy/wooga/gradle/releaseNotesGenerator/utils/ReleaseNoteBody.groovy

```diff
@@ -79,11 +79,13 @@ class ReleaseNoteBody {
 
             def changes = body.readLines().findAll { it.trim().startsWith("* ![") }
             changeList = changes.collect {
-                def match = (it =~ /\!\[(.*?)\] (.*)/)
-                String category = match[0][1].toString()
-                String text = match[0][2].toString()
-                new ChangeNote(category, text)
-            }
+                def match = (it =~ /\!\[(.*?)\] (.+)/)
+                if (match) {
+                    String category = match[0][1].toString()
+                    String text = match[0][2].toString()
+                    new ChangeNote(category, text)
+                }
+            } - null
```

Comment:
What is this?

Reply:
What do you mean with „this“?

Reply:
`- null`

Reply:
Ah this filters the null values that the collect call produces when there is no match

Reply:
Uuhhh that is very ‘groovy’.